pkg-manifest/exportable-manifest/test/index.test.ts

```diff
@@ -91,6 +91,10 @@ test('workspace deps are replaced', async () => {
       baz: 'workspace:baz@^',
       foo: 'workspace:*',
     },
+    peerDependencies: {
+      foo: 'workspace:>= || ^3.9.0',
+      baz: '^1.0.0 || workspace:baz@:>',
```

Comment:
I don't think this syntax should be supported for peer dependencies at all.

Maybe instead of changing the existing function that replaces the spec, just create a new one for peer dependencies specifically.

Reply:
Ya for sure I can do that. Felt weird when I was doing it anyways.

Thanks @zkochan

Reply:
Made the changes @zkochan. I also removed support of `workspace:../` and `workspace:./` as those are also aliasy I believe (let me know if I am incorrect, I can support it easily)